calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,141 @@
+# marker_radar_lidar_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/marker_radar_lidar_calibrator.md)
+
+## Purpose
+
+The package `marker_radar_lidar_calibrator` allows extrinsic calibration between radar and 3d lidar sensors used in autonomous driving and robotics.
+
+Currently, the calibrator only supports the radar that includes distance and azimuth angle, but without elevation angle. For example, ARS 408 radar can be calibrated with this tool. Also, note that the 3d lidar should have a resolution that is high enough to scan several points on the radar reflector (calibration target).
+
+## Inner-workings / Algorithms
+
+The calibrator is designed to accurately predict the transformation between radar and lidar sensors. It starts by pinpointing the central points of reflectors within lidar pointclouds and radar messages, then aligns these points for precise matching. An SVD-based and a yaw-only rotation estimation algorithm are applied to these correlated points to determine the transformation. Specifically, the calibration process consists of four primary steps: constructing a background model, extracting the foreground to detect reflectors, matching and filtering lidar and radar detections, and finally executing the calibration.
+
+### Step 1: Background model construction
+
+Firstly, given the challenge of reliably detecting reflectors, background models for both lidar and radar are constructed from the lidar pointcloud and radar message within a user-defined calibration area, which lacks any calibration targets (such as radar reflectors). More specifically, these background models consist of uniform binary voxel grids that denote whether each voxel represents the background.
+
+### Step 2: Foreground extraction and reflector detection
+
+After the background models for the lidar and radar are established, we extract the foreground points from incoming lidar pointclouds and radar messages that do not align with the background voxels. All foreground radar points are automatically categorized as potential reflector detections. For foreground lidar points, however, the [reflector](#radar-reflector) detection process is more detailed. We first apply a clustering algorithm to identify clusters, then find the highest point in each cluster, and filter the cluster if the highest point is larger than `reflector_max_height`. Next, we average all points within a reflector_radius from the highest point to estimate the center point of the reflector.
```

Comment:
I do not think `align` is the correct word here

Reply:
In the same fashion as the previous section, a section like this should not be chunk of text (split it to make it more readable)

Reply:
`reflector_radius`?

Reply:
"We first apply a clustering algorithm to identify clusters" is redundant

Reply:
separate context :https://github.com/tier4/CalibrationTools/pull/164/commits/57f8ddad3e9c5421ca59c90df572b4ba27dc74d7

others: https://github.com/tier4/CalibrationTools/pull/164/commits/3b945d51e7a7b096d00baef1a319fd296bd0ee83

Reply:
The process is not more detailed. It is either: more convoluted, more complex, or it involves more steps

Reply:
Filtering can go both ways (keep in or out). As such it is better to specify which one is it (in this case I assume you wanted to use filter out)

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/e013b707aa93c7e391cf72932ca9ad5ac612401c

Reply:
larger is not the correct adjective here (you use it to describe the highest point)

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/a2e265936b4319c4f29a951759a000daafca446a